src/plugins/store.js

```diff
@@ -383,18 +372,14 @@ function selectPowerSetting(state, { mode, powerOption }) {
     hotwater: 'hestia/local/stat/hotwatermode',
     humidity: 'hestia/local/stat/humiditymode'
   }
+
   // We don't set a local state for this. Merely report to openhab
   // what we want and it will return all the power states for us.
   console.debug(`[sending] ${topics[mode]}: ${powerOption}`)
   client.publish(topics[mode], powerOption)
 
-  if (state.selectedMode !== 'fan') {
-    if (powerOption == 'Boost') {
-      state.modes[mode].boostEnabled = true;
-    } else {
-      state.modes[mode].boostEnabled = false;
-    }
-  }
+  // Ok I lied, let's eagerly update even though we're going to get an openhab response
+  updateMode(state, mode)(powerOption)
```

Comment:
Note here: This might be unnecessary redundancy, but due to openhab being less responsive on the rpi zero I wanted to make sure the UI had an immediate response to the tap event without waiting for a confirmation in state change from openhab.